### PR TITLE
[JBPM-10141] - Clarify PIM documentation/README.md regarding Flyway DDL auto-creation

### PR DESCRIPTION
**Pull request by afalhambra**
**[JBPM-10141](https://issues.redhat.com/browse/JBPM-10141)**

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>

</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -462,6 +462,8 @@ Reference:
 By default, Process Migration service generates an in-memory database schema automatically.
 You should be able to disable it and manage the schema creation by setting and passing the `quarkus.flyway.migrate-at-start=false` system property.
 
+**Note**: Schema auto-creation for other databases different from H2 in-memory database is not supported due to some Flyway constraints and limitations.
```
Comment (ruromero) [inline README.md]: Can you add more details about where to find the ddl-scripts for all the supported databases?

**Review by ruromero — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (afalhambra) [reply README.md]: done

**Review by ruromero — APPROVED** [review]
much better, thanks

**Review by gmunozfe — APPROVED** [review]
Looks good to me, great work @afalhambra !

**Comment by afalhambra** [comment]
@mareknovotny could you please merge?

backports: 
- 7.67.x https://github.com/kiegroup/process-migration-service/pull/88
- 7.67.x-blue https://github.com/kiegroup/process-migration-service/pull/89

**Pull request merged**